db/migrate/20231005162336_add_address_to_veteran_representative.rb

```diff
@@ -0,0 +1,12 @@
+class AddAddressToVeteranRepresentative < ActiveRecord::Migration[6.1]
+  def change
+    add_column :veteran_representatives, :address_line_1, :string
+    add_column :veteran_representatives, :city, :string
+    add_column :veteran_representatives, :state, :string
+    add_column :veteran_representatives, :zip_code, :string
+    add_column :veteran_representatives, :lat, :float
+    add_column :veteran_representatives, :long, :float
+    add_column :veteran_representatives, :location, :st_point, geographic: true
+    add_column :veteran_representatives, :raw_address, :jsonb
```

Comment:
Context: we will get a file (most likely on a monthly cadence) from OGC that we'll use to add new reps and update work location information. The update task/job will validate the addresses with Lighthouse and get the lat/long from the response.

We only want to store validated data in these columns, but we also don't want to have useless calls to Lighthouse on addresses we know to be invalid. That's why we'll store the raw OGC address and use it to compare to the OGC data in the monthly file to see if the data has changed.

Reply:
So `raw_address` will contain `address_1`, `city`, `state`, `zip_code`?

Reply:
It can. I wanted to give us flexibility here. And with what we know about the OGC data now, we might need to put address_2 in there depending on if there's any cleanup prior to ingesting the file.

Reply:
jsonb == flexibility

Very good.